signer/tests/integration/setup.rs

```diff
@@ -198,6 +210,20 @@ impl TestSweepSetup {
         }
     }
 
+    /// Store all pending transaction packages to the database. Note that this
+    /// is kept separate from [`Self::store_sweep_tx`] because this method
+    /// represents the transaction package which was broadcast to the mempool
+    /// (but not necessarily confirmed), while [`Self::store_sweep_tx`]
+    /// represents the sweep transaction which has been observed on-chain.
+    pub async fn store_sweep_transactions(&mut self, db: &PgStore) -> Vec<model::SweepTransaction> {
```

Comment:
Can we change the name of this or the other fn? Otherwise is kinda confusing which is doing what